### PR TITLE
Basic support for MSSQL server via odbc connection

### PR DESCRIPTION
**Pull request by jnormington**

### CODE REVIEW

File: drivers/mssql.go [hunk 1]
```diff
@@ -0,0 +1,125 @@
...
+	}
+
+	return ms.buildConnString(dc), nil
+}
+
+func (ms mssql) buildConnString(dc *models.DatabaseConfig) string {
+	var buf bytes.Buffer
+	var password string
+
+	conn := "odbc:server={%s};port=%s;user id={%s};%s;database=%s"
```
Comment (vivienschilis) [inline drivers/mssql.go]: Why not moving it to line 63? it doesn't seem to be used other than there?

File: drivers/mssql.go [hunk 2]
```diff
@@ -0,0 +1,125 @@
...
+
+	if dc.Port == "" {
+		dc.Port = mssqlPort
+	}
+
+	// Set the application intent to readonly connection
+	dc.Params["ApplicationIntent"] = "ReadOnly"
+}
+
+func (ms mssql) buildParams(buf *bytes.Buffer, str string) {
```
Comment (vivienschilis) [inline drivers/mssql.go]: Maybe some comments about what this thing do?

**Comment by vivienschilis** [comment]
Also CircleCI

File: drivers/mssql.go (hunk 1 above)
Comment (jnormington) [inline drivers/mssql.go]: Yeah your correct, will update.

File: drivers/mssql.go (hunk 2 above)
Comment (jnormington) [inline drivers/mssql.go]: Sure will add some comment

**Comment by jnormington** [comment]
Thanks @vivienschilis will add codecov and circle ci in a new commit

**Pull request merged**